dm/dm-online-ddl-tool-support.md

```
@@ -23,7 +23,7 @@ MySQLエコシステムでは、gh-ostやpt-oscなどのツールが広く使用
 
 v2.0.5 以降のバージョンでは、 `task`構成ファイル内の`online-ddl`構成項目を使用する必要があります。
 
--   アップストリーム MySQL/MariaDB (同時に) が gh-ost または pt-osc ツールを使用する場合は、タスク構成ファイルで`online-ddl`から`true`設定します。
+-   アップストリーム MySQL/MariaDB (同時に) が gh-ost または pt-osc ツールを使用する場合は、タスク構成ファイルで`online-ddl`から`true`を設定します。
```

markdownlint-cli2 (0.23.2)

[warning] Spaces after list markers
Expected: 1; Actual: 3

(MD030, list-marker-space)